libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSDKCryptoUtils.m

```diff
@@ -27,6 +27,8 @@
 #import <CommonCrypto/CommonCrypto.h>
 #import "NSData+SFAdditions.h"
 #import <Security/Security.h>
+#include "TargetConditionals.h"
```

Comment:
Is this file -> `https://opensource.apple.com/source/CarbonHeaders/CarbonHeaders-18.1/TargetConditionals.h`? Should it be `#import`?